tests/test_remotedesktop.py

```diff
@@ -46,6 +46,19 @@ def test_remote_desktop_create_close_session(self, portals, dbus_con):
         session.close()
         xdp.wait_for(lambda: session.closed)
 
+    def test_remote_desktop_create_session_invalid(self, portals, dbus_con):
+        remotedesktop_intf = xdp.get_portal_iface(dbus_con, "RemoteDesktop")
+
+        request = xdp.Request(dbus_con, remotedesktop_intf)
+        options = {"session_handle_token": "Invalid-token&"}
```

Comment:
Please can we specifically test `"/foo"` and `""` here, too? Those are special cases would be easy to get wrong.

```
for badness in ("Invalid-token&", "/foo", ""):
    options = { "session_handle_token": badness }

    with pytest.raises (etc.)
    ...
```